src/atoms/Spacer/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-import styles from './spacer.module.scss';
+import styles from './spacer.scss';
```

Comment:
Not currently, but there probably should be, with the idea that that dir can just be swapped out to get a different branding across the board.